src/test/java/uk/gov/hmcts/divorce/caseworker/service/CaseworkerIssueApplicationServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.divorce.caseworker.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.divorce.caseworker.service.updater.MiniApplication;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.common.updater.CaseDataContext;
+import uk.gov.hmcts.divorce.common.updater.CaseDataUpdater;
+import uk.gov.hmcts.divorce.common.updater.CaseDataUpdaterChain;
+import uk.gov.hmcts.divorce.common.updater.CaseDataUpdaterChainFactory;
+
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_AUTHORIZATION_TOKEN;
+import static uk.gov.hmcts.divorce.testutil.TestConstants.TEST_CASE_ID;
+import static uk.gov.hmcts.divorce.testutil.TestDataHelper.LOCAL_DATE;
+
+@ExtendWith(MockitoExtension.class)
+class CaseworkerIssueApplicationServiceTest {
```

Comment:
should we include unhappy tests ?

Reply:
I would not add any unhappy path tests here as the only exceptions would be Runtime ones, and they should be tested at in the integration test level

Reply:
yeah makes sense